keyboards/fancytech/fancyalice66/info.json

```diff
@@ -0,0 +1,155 @@
+{
+	"keyboard_name": "FancyAlice66",
+	"manufacturer": "Fancytech",
+	"url": "https://www.aliexpress.com/item/1005004272392611.html",
+	"maintainer": "Adophoxia",
+	"processor": "atmega32u4",
+	"bootloader": "caterina",
```

Comment:
```suggestion
	"development_board": "promicro",
```

Reply:
Board doesn't use a physical Pro Micro and instead uses an embedded MCU.